src/iothread.c

```diff
@@ -348,8 +350,12 @@ void processClientsFromIOThread(IOThread *t) {
             continue;
         }
 
-        /* Update the client in the mem usage */
-        updateClientMemUsageAndBucket(c);
```

Comment:
now we can only update the memory usage once at intervals IO_THREAD_CLIENTS_MAX_CHECK_TIME?
i don't think it's right, please ref the comment:
```
/* Update client memory usage after processing the query buffer, this is
     * important in case the query buffer is big and wasn't drained during
     * the above loop (because of partially sent big commands). */
```